MAINTAINERS.md

```diff
@@ -19,21 +19,21 @@ The following is the full list, alphabetically ordered.
 * Rohit Nayak ([rohit-nayak-ps](https://github.com/rohit-nayak-ps)) rohit@planetscale.com
 * Shlomi Noach ([shlomi-noach](https://github.com/shlomi-noach)) shlomi@planetscale.com
 * Sugu Sougoumarane ([sougou](https://github.com/sougou)) sougou@planetscale.com
-* Vicent Marti ([vmg])(https://github.com/vmg)) vmg@planetscale.com
+* Vicent Marti ([vmg](https://github.com/vmg)) vmg@planetscale.com
 
 ## Areas of expertise
 
 ### General Vitess
-sougou, demmer, rafael, dweitzman, tirsen, askdba, enisoc
+sougou, deepthi, demmer, rafael, dweitzman, tirsen, askdba, enisoc
 
 ### Builds
 dkhenry, shlomi-noach, ajm188, vmg
 
 ### Resharding
-sougou, rafael, tirsen, dweitzman, systay, rohit-nayak-ps
+sougou, rafael, tirsen, dweitzman, systay, rohit-nayak-ps, deepthi
 
 ### Parser
-sougou, dweitzman, deepthi, systay, harshit-gangal
+sougou, dweitzman, systay, harshit-gangal
```

Comment:
@deepthi: I've noticed I've never added mysql to `parser`, even after I rewrote the whole codegen for our parser. I think it would make sense to have me here. 😅 

```suggestion
sougou, dweitzman, systay, harshit-gangal, vmg
```

Reply:
Added and squashed the commits.